### ISSUE TITLE
Audio file upload option

### ISSUE THREAD

**Issue by ennu1** [vishnumenon/ermine-ai#5]
Is there a plan to add audio upload?

**Comment by vishnumenon** [comment]
Nope, not planning to add that functionality for the time being -- though, if you're interested in making a PR, I'm happy to review!
